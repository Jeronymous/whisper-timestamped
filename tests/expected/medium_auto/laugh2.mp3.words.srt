1
00:00:00,180 --> 00:00:00,600
Hehehe

