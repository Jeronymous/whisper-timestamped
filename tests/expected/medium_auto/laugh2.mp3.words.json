{
  "text": " Hehehe",
  "segments": [
    {
      "id": 0,
      "seek": 0,
      "start": 0.22,
      "end": 0.64,
      "text": " Hehehe",
      "tokens": [
        50364,
        634,
        23500,
        50464
      ],
      "temperature": 0.0,
      "avg_logprob": -1.0079832077026367,
      "compression_ratio": 0.5,
      "no_speech_prob": 0.3699853718280792,
      "confidence": 0.334,
      "words": [
        {
          "text": "Hehehe",
          "start": 0.22,
          "end": 0.64,
          "confidence": 0.334
        }
      ]
    }
  ],
  "language": "en",
  "language_probs": {
    "en": 0.6829636693000793,
    "zh": 0.020623717457056046,
    "de": 0.017097661271691322,
    "es": 0.016832586377859116,
    "ru": 0.02336973138153553,
    "ko": 0.09388463199138641,
    "fr": 0.014624404720962048,
    "ja": 0.041015155613422394,
    "pt": 0.01657162234187126,
    "tr": 0.002581360749900341,
    "pl": 0.0035283018369227648,
    "ca": 0.00028962065698578954,
    "nl": 0.0075870417058467865,
    "ar": 0.003366729011759162,
    "sv": 0.0032631459180265665,
    "it": 0.0035838645417243242,
    "id": 0.004898561630398035,
    "hi": 0.004061050247400999,
    "fi": 0.003065441967919469,
    "vi": 0.002663301769644022,
    "he": 0.0002516267995815724,
    "uk": 0.0007872705464251339,
    "el": 0.0005410826415754855,
    "ms": 0.0037558579351752996,
    "cs": 0.0009952031541615725,
    "ro": 0.0006733882473781705,
    "da": 0.00031808606581762433,
    "hu": 0.0006131269037723541,
    "ta": 0.00101087533403188,
    "no": 0.0004020984342787415,
    "th": 0.005214491859078407,
    "ur": 0.0004020984342787415,
    "hr": 0.00012073184188921005,
    "bg": 8.8329310528934e-05,
    "lt": 2.2684840587317012e-05,
    "la": 0.0005163046298548579,
    "mi": 0.0007872705464251339,
    "ml": 0.0007057047332637012,
    "cy": 0.001640812843106687,
    "sk": 6.362121348502114e-05,
    "te": 0.00036043874570168555,
    "fa": 5.441810571937822e-05,
    "lv": 1.4646435374743305e-05,
    "bn": 5.7927773013943806e-05,
    "sr": 1.9514486666594166e-06,
    "az": 7.178978194133379e-07,
    "sl": 0.00010654547804733738,
    "kn": 2.1432470020954497e-06,
    "et": 1.7123416910180822e-05,
    "mk": 7.641982051609375e-07,
    "br": 0.00010654547804733738,
    "eu": 1.272504323424073e-05,
    "is": 3.24945394822862e-05,
    "hy": 8.345286914845929e-06,
    "ne": 2.0334560758783482e-05,
    "mn": 2.304207555425819e-05,
    "bs": 1.088430690288078e-05,
    "kk": 3.5336167911736993e-06,
    "sq": 2.390964937148965e-06,
    "sw": 0.0001265258906641975,
    "gl": 8.8329310528934e-05,
    "mr": 5.825949301652145e-06,
    "pa": 2.751983856796869e-06,
    "si": 0.0001456304162275046,
    "km": 0.0007750650402158499,
    "sn": 0.00042139552533626556,
    "yo": 2.304207555425819e-05,
    "so": 1.9935302475460048e-07,
    "af": 1.1229812116653193e-05,
    "oc": 9.309840606874786e-06,
    "ka": 5.011734742765839e-07,
    "be": 1.5111363609321415e-05,
    "tg": 5.7563557476214555e-08,
    "sd": 2.795321279336349e-06,
    "gu": 5.951582124907873e-07,
    "am": 1.1119018381577916e-06,
    "yi": 4.262356014805846e-06,
    "lo": 2.3773511202307418e-05,
    "uz": 1.9585014499057252e-08,
    "fo": 3.820750862360001e-06,
    "ht": 7.138101864256896e-06,
    "ps": 9.023407073982526e-06,
    "tk": 1.7051567624548625e-07,
    "nn": 0.007706519681960344,
    "mt": 4.563236188914743e-07,
    "sa": 3.798996112891473e-05,
    "lb": 8.64131379785249e-08,
    "my": 2.5306770112365484e-05,
    "bo": 1.4419363651541062e-05,
    "tl": 0.002663301769644022,
    "mg": 4.286763726213394e-07,
    "as": 7.406862891912169e-07,
    "tt": 4.698200939401431e-08,
    "haw": 0.0005326939281076193,
    "ln": 4.6350965021701995e-07,
    "ha": 3.087639868226688e-07,
    "ba": 2.0249235888059047e-07,
    "jw": 0.0017195570981130004,
    "su": 6.045305553925573e-07
  }
}